### PR TITLE
Implement Dense Reward Dictionary v2

### PR DESCRIPTION
**Pull request by Mayukh747**
New PR since the other one already merged.

Your last comment on the previous PR was correct:
`"Should the loose not be -ceil((board_size * board_size)/2) as per the readme..."`

I went ahead and make the fix.

### CODE REVIEW

File: ourhexenv.py [hunk 1]
```diff
@@ -93,8 +93,8 @@ def __init__(self, board_size=5, sparse_flag=True):
             }
         else:
             self.reward_mapping: Dict[str, int] = {
-                "WIN": (self.board_size*self.board_size)//2,
-                "LOSE": -10,
+                "WIN": math.floor((board_size * board_size)/2),
+                "LOSE": math.ceil((board_size * board_size)/2),
```
Comment (devabhixda) [inline ourhexenv.py:97]: It's - of the value

```suggestion
                "LOSE": -math.ceil((board_size * board_size)/2),
```

File: ourhexenv.py [hunk 2]
```diff
@@ -93,8 +93,8 @@ def __init__(self, board_size=5, sparse_flag=True):
             }
         else:
             self.reward_mapping: Dict[str, int] = {
-                "WIN": (self.board_size*self.board_size)//2,
```
Comment (devabhixda) [inline ourhexenv.py:96]: Let's do max_reward = self.board_size*self.board_size)//2

WIN: math.floor(max_reward), Loose: -math.ceil(max_reward)

File: ourhexenv.py (hunk 1 above)
Comment (SaiPrakash2k) [inline ourhexenv.py:97]: A +ve value for losing???

File: ourhexenv.py (hunk 2 above)
Comment (Mayukh747) [reply ourhexenv.py:96]: i like that, coming right up

**Comment by devabhixda** [comment]
LGTM

**Pull request merged**